### PR TITLE
[chores,feat] HM Readme along with conversion tools for manual downloads

### PR DESCRIPTION
**Pull request by apsdehal**
- Adds conversion tool for manual downloads needed in HM
- Adds README for the project
- Add some missing configs under HM
- Add command directly to setup for easy accessibility

### CODE REVIEW

File: projects/hateful_memes/README.md [hunk 1]
```diff
@@ -1 +1,87 @@
 # Hateful Memes Dataset
+
+This folder contains configs required to reproduce results and baselines in the [Hateful Memes challange paper](https://arxiv.org/abs/2005.04790). For details on participation in the challenge, please look at our [Hateful Memes Challenge tutorial](https://mmf.readthedocs.io/en/latest/notes/hateful_memes_challenge.html).
```
Comment (douwekiela) [inline projects/hateful_memes/README.md:3]: s/challange/challenge/

Comment (douwekiela) [reply projects/hateful_memes/README.md:3]: Is it a bit weird to link to that tutorial already if it doesn't exist yet? Can we add this later?

File: projects/hateful_memes/README.md [hunk 2]
```diff
@@ -1 +1,87 @@
...
+
+Please cite the following paper if you use these models and hateful memes dataset:
```
Comment (douwekiela) [inline projects/hateful_memes/README.md:5]: the hateful memes dataset

File: projects/hateful_memes/README.md (hunk 1 above)
Comment (apsdehal) [reply projects/hateful_memes/README.md:3]: Vedanuj is adding this in a separate PR.

File: projects/hateful_memes/README.md [hunk 3]
```diff
@@ -1 +1,111 @@
...
+```
+mmf_run config=<REPLACE_WITH_BASELINE_CONFIG> model=<REPLACE_WITH_MODEL_KEY> dataset=hateful_memes run_type=val
+```
+
+## Predictions for Challenge
+
+For running inference (will generate a csv) on validation set run the following command:
+
+```
+mmf_predict config=<REPLACE_WITH_BASELINE_CONFIG> model=<REPLACE_WITH_MODEL_KEY> dataset=hateful_memes run_type=val evaluation.format=csv
```
Comment (vedanuj) [inline projects/hateful_memes/README.md:74]: `evaluation.predict_file_format=csv`

File: projects/hateful_memes/README.md [hunk 4]
```diff
@@ -1 +1,111 @@
...
+```
+
+For running inference (will generate a csv) on test set run the following command:
+
+```
+mmf_predict config=<REPLACE_WITH_BASELINE_CONFIG> model=<REPLACE_WITH_MODEL_KEY> dataset=hateful_memes run_type=test evaluation.format=csv
```
Comment (vedanuj) [inline projects/hateful_memes/README.md:80]: `evaluation.predict_file_format=csv`

File: projects/hateful_memes/README.md [hunk 5]
```diff
@@ -1 +1,111 @@
...
+
+```
+mmf_run config=<REPLACE_WITH_BASELINE_CONFIG> model=<REPLACE_WITH_MODEL_KEY> dataset=hateful_memes run_type=val checkpoint.resume_zoo=<REPLACE_WITH_PRETRAINED_ZOO_KEY>
+```
+
+To generate predictions using pretrained models, use the following command:
+
+
+```
+mmf_predict config=<REPLACE_WITH_BASELINE_CONFIG> model=<REPLACE_WITH_MODEL_KEY> dataset=hateful_memes run_type=test evaluation.format=csv checkpoint.resume_zoo=<REPLACE_WITH_PRETRAINED_ZOO_KEY>
```
Comment (vedanuj) [inline projects/hateful_memes/README.md:96]: `evaluation.predict_file_format=csv`

**Comment by apsdehal** [comment]
`evaluation.predict_file_format=csv` has been added by default to the hateful memes dataset config.

**Pull request closed without merging**